pkgs/os-specific/linux/firmware/linux-firmware/default.nix

```diff
@@ -7,11 +7,11 @@
 
 stdenvNoCC.mkDerivation rec {
```

Comment:
I am baffled as to why this does not use `finalAttrs` yet, though obviously this is not a blocker.